packages/mroonga-package-task.rb

```diff
@@ -237,18 +237,19 @@ def split_mysql_package
 
   def detect_mysql_community_rpm_version
     series = split_mysql_package[1]
+    # CDN edge nodes may return old cached list without a cache‑buster.
```

Comment:
Could you update the comment why we use no trailing `/` URI?

Reply:
fix: [0e34b86](https://github.com/mroonga/mroonga/pull/915/commits/0e34b86a3834abb9a21a52169535666350a49d64) Thanks!